guides/orders/overview.md

```diff
@@ -0,0 +1,162 @@
+# Overview
+
+The `Spree::Order` model is one of the key models in Solidus. It provides a
+central place around which to collect information about a customer order. It
+collects line items, adjustments, payments, addresses, return authorizations,
+and shipments. 
+
+Whenever information about the order is updated, the `Spree::OrderUpdater`
+should update the order. If your store creates or changes the functionality
+around the order, you may need to [manually call the order
+updater][update-orders].
+
+Orders have the following attributes:
+
+- `number`: The unique identifier for this order. It begins with the letter `R`
+  and ends in a nine-digit number (for example, `R123456789`). This number is
+  shown to the users, and can be used to find the order by calling
+  `Spree::Order.find_by(number: "R123456789")`.
+- `item_total`: The sum of all the line items for this order.
+- `total`: The sum of the `item_total` and the `adjustment_total` attributes.
+- `state`: The current state of the order. See the [Order state
+  machine][order-state-machine] article for more information.
+- `adjustment_total`: The sum of all adjustments on this order.
+- `user_id`: The ID for the order's corresponding user. Stored only if the order
+  is placed by a signed-in user.
+- `completed_at`: The timestamp that logs when the order is completed.
+- `bill_address_id` and `ship_address_id`: The IDs for the related
+  `Spree::Address` objects with billing and shipping address information.
+- `payment_total`: The sum of all the *finalized* payments on the order.
+- `shipment_state`: The current [shipment state][shipment-states] of the order.
+- `payment_state`: The current payment state of the order.
```

Comment:
this state needs a link to its own documentation like `shipment state`, right? I assume it's not here since the payment docs is not merged yet, what about a todo? is it possible within the list?

Reply:
Yes, good call. Thank you!